src/app/background/index.ts

```diff
@@ -36,3 +37,18 @@ const handleConnect = (port: Port) => store.dispatch(connect(port));
 
 browser.runtime.onConnectExternal.addListener(handleConnect);
 browser.runtime.onConnect.addListener(handleConnect);
+
+// https://stackoverflow.com/questions/37000136/check-if-item-is-already-in-the-context-menu
+browser.contextMenus.remove(OPTIONS_MENU_ITEM_ID).then(() => {
```

Comment:
Starting the extension gives me an error on this line in the background : 

![image](https://user-images.githubusercontent.com/3037833/89872846-d52ab500-dbb9-11ea-9161-e033f6f34af0.png)

Maybe we have to either `removeAll` or maybe `try`/`catch` and `create` in the `catch` ?